sonar/modules/documents/api.py

```diff
@@ -298,6 +292,15 @@ def get_files_list(self):
         ]
         return sorted(files, key=lambda file: file.get('order', 100))
 
+    def update(self, data):
+        """Update record.
+
+        :param data: Record data.
+        :returns: Record instance
+        """
+        self.guess_controlled_affiliations(data)
```

Comment:
perhaps this can be private `_`....

Reply:
This is a class method, bad idea to flag it private, no?